src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -54,21 +54,20 @@ for further details.
  * Sidebar + main containers
  */
 
+/* Note: container is components/mobile-menu-container */
```

Comment:
Have slightly refactored layout here, since otherwise `docsVersionSwitcherWrapper` would overflow onto footer (as seen in screen recording attached to [this task](https://app.asana.com/0/1202097197789424/1202416702159968/f)).